helpers/database-import-scripts/rnacentral/generate_rnacentral_json.py

```diff
@@ -372,9 +410,12 @@ def get_project_accession(biosample):
     if r.ok:
         projects = list()
         elements = r.text.split("\n")
-        for e in elements:
-            if not e.startswith("run_accession") and not e == "":
-                projects.append(e.strip().split("\t")[1])
+        header = elements[0]
+        col_names = header.strip().split("\t")
+        study_index = col_names.index("secondary_study_accession")
+        for e in elements[1:]:
+            if not e == "":
+                projects.append(e.strip().split("\t")[study_index])
```

Comment:
A few asserts here should help catch unexpected values

Reply:
Done